File: general-client/src/main/java/com/song/middleware/client/util/CollectionUtils.java
package com.song.middleware.client.util;

import java.util.Collection;

/**
 * Created by song on 2017/8/6.
 */
public class CollectionUtils {

    public static boolean isEmpty(Collection collection) {
        return collection == null || collection.isEmpty();
    }

    public static boolean isNotEmpty(Collection collection) {
        return !isEmpty(collection);
    }

}

File: general-client/src/main/java/com/song/middleware/client/util/CollectionUtils.kt
package com.song.middleware.client.util

/**
 * Created by song on 2017/8/6.
 */
object CollectionUtils {

    fun isEmpty(collection: Collection<*>?): Boolean {
        return collection == null || collection.isEmpty()
    }

    fun isNotEmpty(collection: Collection<*>): Boolean {
        return !isEmpty(collection)
    }

}
